test/metabase/driver/sql_jdbc/connection_test.clj

```diff
@@ -133,7 +147,7 @@
                     ;; even after getting the latest `DatabaseInstance`
                     (sync/sync-database! db {:scan :schema})
                     (is (some? pool-spec-2))
-                    (is (= 1 @hash-change-called-times))
+                    (is (= 1 @hash-change-called-times) "One hash change should have been logged")
```

Comment:
Added context because otherwise a test failure like `(not= 1 0)` is a little bit confusing to read.